rot/cipher/Cipher.java

```diff
@@ -2,6 +2,17 @@
 import java.lang.StringBuilder;
 import java.util.LinkedList;
 
+abstract class Cipher {
+	private final CharacterMap characaterMap;
+
+	public Cipher(int rot) {
+	characterMap = new CharacterMap(rot);
+	}
+
+	abstract public String translateText() {
```

Comment:
public abstract...

Reply:
ooo... This is cool, I didnt know you can review my code and comment it this way, I have to learn more about github too.
Thanks for your comment, I wrote it in the right order in the last pushed version.